packages/driver/src/cy/chai/inspect.js

```diff
@@ -0,0 +1,427 @@
+// This is (almost) directly from Node.js utils
```

Comment:
if its *almost* directly from Node.js utils then we should probably indicate what we are changing that's not the same thing. ideally we can find another way to accomplish this via monkeypatching or something else to avoid this bigass file